mingw-w64+gcc.sh

```diff
@@ -48,6 +64,8 @@ cd build
 ../src/configure --enable-languages=c,c++ --build=x86_64-w64-mingw32 --host=x86_64-w64-mingw32 \
 --target=x86_64-w64-mingw32 --disable-multilib --prefix=/c/temp/gcc/dest --with-sysroot=/c/temp/gcc/dest \
 --disable-libstdcxx-pch --disable-libstdcxx-verbose --disable-nls --disable-shared --disable-win32-registry \
+--enable-libatomic --enable-threads=posix \
+--enable-lto --enable-libgomp \
```

Comment:
Is LTO completely unrelated? The last time I checked, which was recently, LTO could be enabled but `-flto` immediately triggered silent bad codegen.